preprocessing/nextclade/src/loculus_preprocessing/processing_functions.py

```diff
@@ -315,15 +302,15 @@ def parse_timestamp(
             )
             return ProcessingResult(
                 datum=None,
-                warnings=[
+                errors=[
```

Comment:
Is this behaviour change intentional? It's not mentioned in the PR description so it might be accidental? We should discuss when we change something from error -> warning - maybe make such logic changes in a separate PR that's more targeted and unrelated to testing.

Reply:
It is intentional - I should mention it in the PR description. Everywhere else we now raise an error if e.g. the date cannot be parsed or the int/float/bool is invalid - so changing this to an error is more in line with the rest of the code.

Reply:
Ah good! It's also not used by human input, but by ingest input only, so warning doesn't make sense anyways! Thanks!

https://github.com/pathoplexus/pathoplexus/blob/b7994aec38b4e15c3e956d3dd8ecaae93b02b299/loculus_values/values.yaml#L95-L114

<img width="630" alt="Brave Browser 2024-10-08 10 40 59" src="https://github.com/user-attachments/assets/f2804642-1e2d-44cb-a2ac-7155f53e6997">